code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -304,13 +304,14 @@
 	reagent_state = SOLID
 	color = "#000067" // rgb: 0, 0, 103
 	toxpwr = 0
-	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+	metabolization_rate = 1 * REAGENTS_METABOLISM
 
 /datum/reagent/toxin/chloralhydratedelayed/on_mob_life(mob/living/carbon/M)
 	switch(current_cycle)
 		if(10 to 20)
 			M.confused += 1
 			M.drowsyness += 1
+			M.adjustStaminaLossBuffered(5)
```

Comment:
Affecting the stamina buffer directly is a hard no, as the stamina buffer is intended to only be affected by the user's actions and only the user's actions.